### PR TITLE
feat(accordion): initial version

### PR DESCRIPTION
**Pull request by ValentinNelu**

### CODE REVIEW

File: page-objects/lib/accordion.po.ts [hunk 1]
```diff
@@ -0,0 +1,93 @@
+import type {Locator} from '@playwright/test';
+import {BasePO} from '../../base-po/lib/base.po';
```
Comment (divdavem) [inline page-objects/lib/accordion.po.ts:2]:
```suggestion
import {BasePO} from '@agnos-ui/base-po';
```

File: e2e/accordion/accordion.e2e-spec.ts [hunk 2]
```diff
@@ -0,0 +1,62 @@
+import {expect, test} from '@playwright/test';
+import {AccordionPO} from '../../page-objects/lib';
```
Comment (divdavem) [inline e2e/accordion/accordion.e2e-spec.ts:2]:
```suggestion
import {AccordionPO} from '@agnos-ui/page-objects';
```

File: page-objects/lib/accordion.po.ts (hunk 1 above)
Comment (quentinderoubaix) [reply page-objects/lib/accordion.po.ts]: was fixed.

File: core/lib/accordion.ts [hunk 3]
```diff
@@ -0,0 +1,434 @@
...
+	});
+	const itemTransition = createTransition({
+		transition: itemTransition$,
+		visible: computed(() => {
+			const collapsed = itemCollapsed$();
+			onItemCollapsedChange$()(collapsed);
+			return !collapsed;
+		}),
+		animation: itemAnimation$,
+		animationOnInit: readable(false),
```
[the remark this answers is not in the document]
Comment (quentinderoubaix) [reply core/lib/accordion.ts]: was fixed

File: e2e/accordion/accordion.e2e-spec.ts (hunk 2 above)
Comment (quentinderoubaix) [reply e2e/accordion/accordion.e2e-spec.ts]: was fixed

File: e2e/demo-po/accordion.po.ts [hunk 4]
```diff
@@ -0,0 +1,8 @@
+import {BasePO} from '../../base-po/lib/base.po';
```
[the remark this answers is not in the document]
Comment (quentinderoubaix) [reply e2e/demo-po/accordion.po.ts]: was fixed

File: angular/demo/src/app/samples/accordion/default.route.ts [hunk 5]
```diff
@@ -0,0 +1,45 @@
+import {AgnosUIAngularModule} from '@agnos-ui/angular';
+import {NgFor} from '@angular/common';
+import {Component} from '@angular/core';
+
+@Component({
+	standalone: true,
+	imports: [AgnosUIAngularModule, NgFor],
```
Comment (quentinderoubaix) [inline angular/demo/src/app/samples/accordion/default.route.ts:7]: unnecessary NgFor

File: angular/lib/src/lib/accordion/accordion.component.ts [hunk 6]
```diff
@@ -0,0 +1,266 @@
...
+	},
+	hostDirectives: [
+		{
+			directive: UseDirective,
+		},
+	],
+	imports: [SlotDirective, UseDirective],
+	template: ` <ng-template [auSlotProps]="{state: state(), widget}" [auSlot]="state().slotItemStructure"></ng-template> `,
+})
+export class AccordionItemComponent implements OnChanges, AfterContentChecked, AfterViewInit {
```
Comment (quentinderoubaix) [inline angular/lib/src/lib/accordion/accordion.component.ts:124]: AccordionItem should probably have ```OnPush``` as change detection strategy

File: react/demo/app/samples/accordion/Default.route.tsx [hunk 7]
```diff
@@ -0,0 +1,29 @@
+import {Accordion as AccordionAgnosUI, AccordionItem as AccordionItemAgnosUI} from '@agnos-ui/react';
```
Comment (quentinderoubaix) [inline react/demo/app/samples/accordion/Default.route.tsx:1]: No need to rename the imports, keep Accordion and AccordionItem

**Pull request merged**